examples/facebook_bot.js

```diff
@@ -488,6 +488,11 @@ controller.hears(['send tagged message'], 'message_received', function (bot, mes
     bot.reply(message, taggedMessage);
 });
 
+controller.hears(['insights'], 'message_received', function (bot, message) {
+    controller.api.insights.get_insights(['page_messages_active_threads_unique', 'page_messages_blocked_conversations_unique'], null, null, function (err, body) {
+        bot.reply(message, JSON.stringify(body.data));
```

Comment:
A little bit of error handling would be nice here, especially because many people use examples with little modification.
body can be undefined, if that's the case then accessing body.data would throw an error.

```js
if (err) {
    bot.reply(message, 'Insights error');
} else {
    bot.reply(message, JSON.stringify(body.data));
}
```

Reply:
It looks nice, done 👍